desktop/apps/article2/components/App.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'underscore'
+import components from '@artsy/reaction-force/dist/components/publishing/index'
+import moment from 'moment'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+const { ImagesetPreview, Artwork, Image, FeatureHeader } = components
+
+export default class App extends Component {
+  static propTypes = {
+    article: PropTypes.object
+  }
+
+  renderSections (sections) {
+    const renderedSections = sections.map((section) => {
+      switch (section.type) {
+        case 'image_collection':
+          const images = section.images.map((image) => {
+            if (image.type === 'image') {
+              return (
+                <div style={{ width: '50%', margin: 'auto' }}>
+                  <Image image={image} />
+                </div>
+              )
+            } else if (image.type === 'artwork') {
```

Comment:
This just displays the images in a block, not with fillwidth yet. I figured that'd be more sensible to do on the Reaction side?